cmd/shield/main.go

```diff
@@ -159,6 +151,78 @@ func main() {
 	c.Alias("stat", "status")
 
 	/*
+	   ########     ###     ######  ##    ## ######## ##    ## ########   ######
+	   ##     ##   ## ##   ##    ## ##   ##  ##       ###   ## ##     ## ##    ##
+	   ##     ##  ##   ##  ##       ##  ##   ##       ####  ## ##     ## ##
+	   ########  ##     ## ##       #####    ######   ## ## ## ##     ##  ######
+	   ##     ## ######### ##       ##  ##   ##       ##  #### ##     ##       ##
+	   ##     ## ##     ## ##    ## ##   ##  ##       ##   ### ##     ## ##    ##
+	   ########  ##     ##  ######  ##    ## ######## ##    ## ########   ######
+	*/
+	c.HelpBreak()
+	c.HelpGroup("BACKENDS:")
+	c.Dispatch("backends", "List configured SHIELD backends",
+		func(opts Options, args []string) error {
+			DEBUG("running 'backends' command")
+
+			var indices []string
+			for k, _ := range Cfg.Aliases {
+				indices = append(indices, k)
+			}
+			sort.Strings(indices)
+
+			t := tui.NewTable("Name", "Backend URI")
+			for _, alias := range indices {
+				be := map[string]string{"name": alias, "uri": Cfg.Aliases[alias]}
+				t.Row(be, be["name"], be["uri"])
+			}
+			t.Output(os.Stdout)
+
+			return nil
+		})
+	c.Alias("list backends", "backends")
+	c.Alias("ls be", "backends")
+
+	c.Dispatch("create backend", "Create Backend",
+		func(opts Options, args []string) error {
+			DEBUG("running 'create backend' command")
+
+			if len(args) != 2 {
+				return fmt.Errorf("Invalid 'create backend' syntax: `shield backend <name> <uri>")
+			}
+			err := Cfg.AddBackend(args[1], args[0])
+			if err != nil {
+				return err
+			}
+			Cfg.Save()
+
+			ansi.Fprintf(os.Stdout, "Successfully created backend '@G{%s}', pointing to '@G{%s}'\n\n", args[0], args[1])
```

Comment:
Can we implicitly use the backend after we create it?  To avoid the following churn:

```
shield create backend prod http://prod.shield.us
shield use backend prod
```